dev/tasks/matlab/github.yml

```
@@ -59,8 +59,14 @@ jobs:
           path: matlab-arrow-ubuntu.tar.gz
 
   macos:
-    name: AMD64 macOS 12 MATLAB
-    runs-on: macos-12
+    {% set arch = '${{ matrix.platform.architecture}}' %}
+    name: {{ arch }} macOS MATLAB
```

It seems that `arch` is only used here. Can we simplify this?

```suggestion
    name: {{ '${{ matrix.platform.architecture}}' }} macOS MATLAB
```

---

Will do!